### PR TITLE
MVP-350 capabilities learn how to export carousel

### PR DESCRIPTION
**Pull request by jestanoff**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

Adds templates and CSS for capabilities how to export carousel. These are still not rendered under any page, should wired up in a follow up PR from BE

### CODE REVIEW

File: learn/templates/learn/how_to_export_second_slide.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "core/base.html" %}
+
+{% block content %}
+    <section id="learn-root">
+        <div class="learn__wrapper">
+            <div class="learn__carousel">">
+                <p class="learn__carousel--paragraph">Develop your skills in exporting with our interactive online training, that helps you create your export plan, as you learn.</p>
+                <nav class="learn__carousel-nav">
+                    <ul class="learn__carousel-ul">
+                        <li class="learn__nav-circle"><a href="/learn/how-to-export-first-slide">Go to slide 1</a></li>
```
Comment (sdonk) [inline learn/templates/learn/how_to_export_second_slide.html:10]: page_url tag?

File: learn/templates/learn/how_to_export_second_slide.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+                        <li class="learn__nav-circle--active learn__nav-circle"><a href="/learn/how-to-export-second-slide">Go to slide 2</a></li>
+                        <li class="learn__nav-circle"><a href="/learn/how-to-export-third-slide">Go to slide 3</a></li>
```
Comment (sdonk) [inline learn/templates/learn/how_to_export_second_slide.html:12]: ditto

File: learn/templates/learn/how_to_export_second_slide.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+                    </ul>
+                    <a class="button" href="/learn/how-to-export-second-step">Continue</a>
```
Comment (sdonk) [inline learn/templates/learn/how_to_export_second_slide.html:14]: ditto

File: learn/templates/learn/how_to_export_second_slide.html (hunk 1 above)
Comment (jestanoff) [reply learn/templates/learn/how_to_export_second_slide.html:10]: As discussed I'm waiting until these pages are created in another PR, so I can hook these here

**Pull request merged**